apps/ledger-live-mobile/e2e/models/liveApps/liveAppWebview.ts

```diff
@@ -11,7 +11,7 @@ export default class LiveAppWebview {
 
   async startLiveApp(liveAppDirectory: string, liveAppPort = 3000) {
     try {
-      const port = await startDummyServer(`${liveAppDirectory}/build`, liveAppPort);
+      const port = await startDummyServer(`${liveAppDirectory}/dist`, liveAppPort);
```

Comment:
@Justkant should we wait before merging then? 👀

Reply:
Yes, we are starting dummy-wallet-app in this test. It should be OK, right ?
https://github.com/LedgerHQ/ledger-live/blob/d0f9a7c62523e2f83d680f18240b2f3d8a7ece38/apps/ledger-live-mobile/e2e/specs/wallet-api.spec.ts#L25

Reply:
@abdurrahman-ledger You are correct, go for your current solution, I was wondering if mobile was testing other dummy apps but from what I saw it's only testing the wallet-api on LLM

@gre Don't wait for me because I'm not on the subject currently as we have other priorities in the meantime and won't impact this fix